src/bioregistry/data/bioregistry.json

```diff
@@ -14869,7 +14869,7 @@
     },
     "uri_format": "http://purl.obolibrary.org/obo/CHEBI_$1",
     "wikidata": {
-      "description": "identifier from database and ontology of molecular entities focused on 'small' chemical compounds",
+      "description": "identifier from database and ontology of molecular entities focused on 'small' chemical compounds",
```

Comment:
I'm not sure what happened here but there seem to be spurious diffs related to spaces. @tanayshah2 do you know how these were introduced? It's possible that they are changed automatically by an editor you used to edit and save the JSON file. Ideally we should just edit the URLs in question and try to keep everything else unchanged.